### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by cychub**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -147,12 +147,12 @@ or the `npm_config_sharp_local_prebuilds` environment variable.
...
 if `sharp_binary_host` is set to `https://hostname/path`
 and the sharp version is `1.2.3` then the resultant URL will be
-`https://hostname/path/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
+`https://hostname/path/v1.2.3/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
 
 Filename example:
 if `sharp_local_prebuilds` is set to `/path`
 and the sharp version is `1.2.3` then the resultant filename will be
-`/path/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
+`/path/v1.2.3/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
```
Comment (lovell) [inline docs/install.md:155]: The local prebuilds feature of prebuild-install doesn't add the version-as-a-path-component, so please can you remove the change on this line.

https://github.com/prebuild/prebuild-install#local-prebuilds

File: docs/install.md [hunk 2]
```diff
@@ -147,12 +147,12 @@ or the `npm_config_sharp_local_prebuilds` environment variable.
 URL example:
 if `sharp_binary_host` is set to `https://hostname/path`
 and the sharp version is `1.2.3` then the resultant URL will be
-`https://hostname/path/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
+`https://hostname/path/v1.2.3/sharp-v1.2.3-napi-v5-platform-arch.tar.gz`.
```
Comment (lovell) [inline docs/install.md:150]: Good spot, thank you.

**Review by lovell — CHANGES_REQUESTED** [review]

File: docs/install.md (hunk 2 above)
Comment (cychub) [reply docs/install.md:150]: updated

File: docs/install.md (hunk 1 above)
Comment (lovell) [reply docs/install.md:155]: Thanks for the update, it looks like this line change is still present so marking as unresolved.

**Comment by lovell** [comment]
Thank you very much!

**Pull request merged**